calc.html

```diff
@@ -1,91 +1,82 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <title>Worlds Best Title Tag</title>
     <!-- from https://www.bootstrapcdn.com/ -->
     <!--- see also https://getbootstrap.com/docs/4.0/getting-started/download/ -->
-    <link
-        href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
-        rel="stylesheet"
-        integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
-        crossorigin="anonymous"
-    >
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <script>
         /**
          * `calculator` is a variable that is an object. The {} brackets denote the variable is an object.
          * `add`, `subtract` and `multiply` are functions. They are also properties of the `calculator` object.
          */
         calculator = {
-            add: function () {
+            add: function() {
                 // we have a bug here due to JS using "+" for addition and concatonation
                 // see https://www.w3schools.com/js/tryit.asp?filename=tryjs_oper_concat5
                 // and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat
                 document.forms.inputs.result.value = document.forms.inputs.number1.value + document.forms.inputs.number2.value;
             },
-            subtract: function () {
+            subtract: function() {
                 document.forms.inputs.result.value = document.forms.inputs.number1.value - document.forms.inputs.number2.value;
             },
-            multiply: function () {
+            multiply: function() {
                 // @TODO: complete this code so it returns the correct result
-                document.forms.inputs.result.value = "We're working on this feature";
+                document.forms.inputs.result.value = document.forms.inputs.number1.value - document.forms.inputs.number2.value;
+            },
+            division: function() {
+                // @TODO: complete this code so it returns the correct result
+                document.forms.inputs.result.value = document.forms.inputs.number1.value - document.forms.inputs.number2.value;
```

Comment:
Your functions are very close to being correct. However in your multiply and divide functions you forgot to replace the '-' with the '*' for multiplication and '/' for division, respectively, after copying and pasting your subtraction function. Additionally, you will find that without wrapping every "document.forms.inputs.number.value" to the right of the equal sign in the parseInt() function your addition function won't work properly and will cause 4 + 3 to result in 43, for example.